app/Models/Missions/Mission.php

```diff
@@ -469,14 +469,7 @@ public function exportedName($format = 'pbo')
     {
         $revisions = $this->revisions()->count();
 
-        $download = 'ARC_' .
-            strtoupper($this->mode == 'adversarial' ? 'tvt' : $this->mode) . '_' .
-            studly_case($this->display_name) . '_' .
-            trim(substr($this->user->username, 0, 4)) . '_' .
-            $revisions . '.' .
-            $this->map->class_name . '.' . $format;
-
-        return $download;
+        return $this->original["file_name"] . '.' . $revisions . '.' . $format;
```

Comment:
Do you have an example of what this new format looks like?

Do you need to do something with the existing files stored in GCS or will this work with them?